data/cncf.yaml

```diff
@@ -671,6 +671,20 @@
       url: https://github.com/etcd-io/zetcd
       check_sets:
         - code-lite
+- name: external-dns
+  display_name: ExternalDNS
+  description: Configure external DNS servers (AWS Route53, Google CloudDNS and others) for Kubernetes Ingresses and Services
+  category: orchestration
+  logo_url: https://github.com/kubernetes-sigs/external-dns/raw/master/docs/img/external-dns.png
+  devstats_url: https://k8s.devstats.cncf.io
```

Comment:
Could you please remove the `devstats_url`, `accepted_at` and `maturity` fields? We are not using them for the K8S subprojects.

Reply:
Done!